nexus/db-queries/src/db/datastore/switch_port.rs

```diff
@@ -1389,13 +1392,12 @@ async fn do_switch_port_settings_delete(
             .returning(SwitchPortLinkConfig::as_returning())
             .get_results_async(conn)
             .await?;
-
     // delete lldp configs
-    use db::schema::lldp_service_config::{self, dsl as lldp_config_dsl};
-    let lldp_svc_ids: Vec<Uuid> =
-        links.iter().map(|link| link.lldp_service_config_id).collect();
-    diesel::delete(lldp_config_dsl::lldp_service_config)
-        .filter(lldp_service_config::id.eq_any(lldp_svc_ids))
+    use db::schema::lldp_link_config;
+    let lldp_link_ids: Vec<Uuid> =
+        links.iter().map(|link| link.lldp_link_config_id).collect();
+    diesel::delete(lldp_link_config::dsl::lldp_link_config)
+        .filter(lldp_link_config::id.eq_any(lldp_link_ids))
```

Comment:
Not a huge deal, but out of curiosity -- `lldp_link_config` has a `time_deleted` field, but it looks like we're doing hard deletion here instead of soft-deletion. When do you plan on using soft vs hard deletion for this table?

Reply:
We don't actually have a way to delete the LLDP config at all - other than deleting the whole port, I guess.  The soft delete field was carried over from the earlier, never-used, tables.  I'm not really sure why we would use a soft delete for this.  I'll see if Ry had a reason for going this way initially, but I expect I'll take it out.